### PR TITLE
Styling components

### PR DESCRIPTION
**Pull request by FevenSeyfu**
In this PR I have completed the below requirements:

- Add any missing JSX elements to your project (without adding extra functionality)

- Your final design should match Zeplin's design (this includes all elements + the same styling)

- You may use the CSS rules detailed in Zeplin to style the website.

### CODE REVIEW

File: src/components/NavBar.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ function NavBar() {
         <li className="NavItem"><Link className="NavLink" to="/">Books</Link></li>
         <li className="NavItem"><Link className="NavLink" to="/catagories">Catagories</Link></li>
       </ul>
-      <div><img src="../assets/avatar.png" style={{ width: '30px', height: '30px' }} alt="Avatar" /></div>
+      <img width="48" height="48" src="https://img.icons8.com/color/48/test-account.png" alt="Avatar" className="Mask" />
```
Comment (Ismailco) [inline src/components/NavBar.js:11]: - [ ] Kindly make sure to space all the elements on your page correctly, you can move the profile icon more to the right and align the component in the center of the page

Screenshot_20230520_183637

File: src/App.js [hunk 2]
```diff
@@ -3,6 +3,7 @@ import Books from './components/Books';
 import Catagories from './components/Catagories';
 import NotFound from './components/NotFound';
 import NavBar from './components/NavBar';
+import './App.css';
 
 function App() {
```
Comment (Ismailco) [inline src/App.js:8]: - [ ] Kindlt consider adding some padding/margin between the `hr` and the book container, also make sure to add space between the `Add New Book` form and the `hr` so your design looks identical to the provided design
- [ ] Kindly make sure that the Books cards and the form is centred on the page and have the proper width

Screenshot_20230520_183925

File: src/components/BookCard.js [hunk 3]
```diff
@@ -20,25 +20,34 @@ const BookCard = () => {
...
+                dispatch(removeBook(book.item_id));
+              }}
+            >
+              Remove
+            </button>
+            <li className="action last">Edit</li>
+          </ul>
+        </div>
+        <div className="progress-chart">
+          <div className="progress-bar"> </div>
```
Comment (Ismailco) [inline src/components/BookCard.js]: - [ ] Kindly consider using a circle for the progress bar/circle so your app respects the provided design rules

Screenshot_20230520_184711

**Review by Ismailco — CHANGES_REQUESTED** [review]
### Hi @FevenSeyfu, 

Good job so far 👍🏼 
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Used Gitflow
✔️ Work documented professionally

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/NavBar.js (hunk 1 above)
Comment (FevenSeyfu) [reply src/components/NavBar.js:11]: Thank you for the feedback.I have added space between the cards to match the design. I have aligned the avatar to the left,

File: src/App.js (hunk 2 above)
Comment (FevenSeyfu) [reply src/App.js:8]: thank you for the feedback I have fixed the progress bar to be circle and aligned the form to match the bookcard.

File: src/components/BookCard.js (hunk 3 above)
Comment (FevenSeyfu) [reply src/components/BookCard.js]: Thank you for the feedback done.

**Review by iambenkis — APPROVED** [review]
## Hi @FevenSeyfu ,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
### Highlights
- Descriptive pull request message with screenshots ✔️
- Well organized project ✔️
approval


# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @iambenkis in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**